3rdParty/LightPcapNg/LightPcapNg/src/light_pcapng_ext.c

```diff
@@ -422,7 +422,7 @@ int light_get_next_packet(light_pcapng_t *pcapng, light_packet_header *packet_he
 			timestamp = timestamp << 32;
 			timestamp += epb->timestamp_low;
 			uint64_t ticks_per_sec = pcapng->file_info->timestamp_ticks_per_second[epb->interface_id];
-			uint64_t packet_secs = timestamp / ticks_per_sec;
+			uint64_t packet_secs = (ticks_per_sec != 0 ?  timestamp / ticks_per_sec : 0);
```

Comment:
also L430

Reply:
Yes, I noticed it later... I added another commit with a fix: https://github.com/seladb/PcapPlusPlus/pull/1535/commits/cde15ce83c5fe8c34b855154935c4be50984011b